styles.css

```diff
@@ -112,15 +128,26 @@ a {
   height: 25px;
 }
 
-.cards-container {
+/* .cards-container {
   background-color: #e5e5e5;
   padding: 11rem 1.5rem;
   display: grid;
   grid-template-rows: repeat(3, 1fr);
   gap: 12rem;
```

Comment:
- [ ] Kindly remove all unused/commented code in this file. Removing such codes makes your code look clean. 👍